lib/telemetry_registry.ex

```diff
@@ -49,38 +50,37 @@ defmodule TelemetryRegistry do
   ```
 
   #### Elixir
-  
+
   Elixir does not allow for declaring a custom attribute multiple times by default. We have included macros
   to help with this and to provide a way to include event documentation.
 
   ```elixir
   defmodule TestElixirApp do
     use TelemetryRegistry
 
-    telemetry_event(%{
+    telemetry_event %{
       event: [:test_elixir_app, :single, :event],
       description: "emitted when this event happens",
       measurements: "%{duration: non_neg_integer()}",
       metadata: "%{status: status(), name: String.t()}"
-    })
+    }
 
-    telemetry_event([:test_elixir_app, :event, :stop])
-  end
-  ```
+    telemetry_event [:test_elixir_app, :event, :stop]
 
-  Add `use TelemetryRegistry` at the top of your module to prep your module for defining events. This
-  handles setting up everything needed to declare events and the very helpful `telemetry_event/1`
-  macro.
+  @moduledoc \"""
```

Comment:
Protip! Use `~S'''` in the parent @doc/@moduledoc so in here you can use `@moduledoc """` without escaping!